### PR TITLE
Include header-includes and preamble for arxiv template; add some ins…

### PR DESCRIPTION
**Pull request by slemonide**
…tructions on how to install this package from a local folder (for development/debug/etc)

See issue #345

### CODE REVIEW

File: README.md
Comment (cderv) [inline README.md:22]: This is not specific to **rticles** - this is development practice. You can also use `install.packages` or RStudio IDE included  build & install feature. 

I don't think we should add that here. Why do you suggest that ?

File: inst/rmarkdown/templates/arxiv/resources/template.tex
Comment (cderv) [inline inst/rmarkdown/templates/arxiv/resources/template.tex:75]: Why add a preamble variable ? 
`header-includes` variable is used for that already. Using `includes` argument in the yaml header for example.
https://bookdown.org/yihui/rmarkdown/pdf-document.html#includes-1

**Review by cderv — CHANGES_REQUESTED** [review]

File: README.md
Comment (slemonide) [reply README.md:22]: I think it's good to leave a note on how to modify and install a local development version. This would make it easier for people to contribute. Maybe move this under "Development" or "Contribute" section?

File: inst/rmarkdown/templates/arxiv/resources/template.tex
Comment (slemonide) [reply inst/rmarkdown/templates/arxiv/resources/template.tex:75]: This makes it consistent with other templates.

Also people expect for `preamble` to work: https://github.com/rstudio/rticles/issues/257#issuecomment-564304252

File: README.md
Comment (cderv) [reply README.md:22]: It we keep something like that it should be moved to contribute section yes. I would prefer to add a link to some documentation about how to work and develop r package like https://r-pkgs.org/index.html instead of developing  a new part on how to develop.

File: inst/rmarkdown/templates/arxiv/resources/template.tex
Comment (cderv) [reply inst/rmarkdown/templates/arxiv/resources/template.tex:75]: Oh I see. so this is a specificity of some **rticles** templates. Not all have it, and it won't work with **rmarkdown**, so it could be expected but by some **rticles** user only. 

Fine by me. Thank you for pointing this out!

File: README.md
Comment (yihui) [reply README.md:22]: I'd also prefer leaving this out. As @cderv said, this is development practice and not specific to the **rticles** package. There are several ways to install a local package, and `remotes::install_local()` is one of them. RStudio users will probably find `Ctrl/Cmd + Shift + B` easier.

**Review by yihui — CHANGES_REQUESTED** [review]
I think this PR is good to merge after documenting the `preamble` option and leaving out the changes in README.md (if you feel it is really necessary, please feel free to submit a separate PR, so it won't block this PR from being merged). Thanks!

File: inst/rmarkdown/templates/arxiv/resources/template.tex
Comment (cderv) [inline inst/rmarkdown/templates/arxiv/resources/template.tex:74]: `preamble` and `header-includes` are the same IMO. I don't think a template should have both - some of existing template have both but not sure why. 

Why do you think we need both ? 

I would rather only add `header-includes` (which #356 also suggest).

**Comment by cderv** [comment]
The `header-include` variable has been added to the template already in https://github.com/rstudio/rticles/pull/356 

I'll close this PR then.

**Pull request closed without merging**
